app/uk/gov/hmrc/incometaxproperty/services/PropertyService.scala

```diff
@@ -111,4 +113,9 @@ class PropertyService @Inject()(connector: IntegrationFrameworkConnector)
       Left(DataNotFoundError)
     }
   }
+
+  def persistAnswers[A](ctx: JourneyContext, answers: A)(implicit
```

Comment:
So, is our pipeline between the levels gonna be Future, or Future[Either[...]]
We might want to define a standard, and have the same structure on all levels. Thanks